internal/backend/crypto/gpg/cli/encrypt.go

```diff
@@ -25,25 +28,32 @@ func (g *GPG) Encrypt(ctx context.Context, plaintext []byte, recipients []string
 		args = append(args, "--trust-model=always")
 	}
 
+	debug.Log("recipients are %s", recipients)
+	var badRecipients []string
 	for _, r := range recipients {
 		kl, err := g.listKeys(ctx, "public", r)
 		if err != nil {
 			debug.Log("Failed to check key %s. Adding anyway. %s", err)
 		} else if len(kl.UseableKeys(gpg.IsAlwaysTrust(ctx))) < 1 {
-			out.Printf(ctx, "Not using invalid key %s for encryption. (Check its expiration date or its encryption capabilities.)", r)
-
+			badRecipients = append(badRecipients, r)
+			errmsg := fmt.Sprintf("Not using invalid key %s for encryption. (Check its expiration date or its encryption capabilities.)", r)
+			debug.Log(errmsg)
+			out.Printf(ctx, errmsg)
 			continue
 		}
+		debug.Log("adding recipient %s", r)
 		args = append(args, "--recipient", r)
 	}
-
 	buf := &bytes.Buffer{}
+	if len(badRecipients) == len(recipients) {
+		return buf.Bytes(), errors.New("no trustable recipients were found!")
```

Comment:
```suggestion
		return buf.Bytes(), errors.New("no usable recipients were found!")
```

Is this necessarily untrusted recipients?